ci-operator/config/openshift/dedicated-admin-operator/openshift-dedicated-admin-operator-master.yaml

```diff
@@ -0,0 +1,28 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: origin-v4.0
+    namespace: openshift
+    tag: base
```

Comment:
Most images now use this as base, are you sure you should use the `openshift/origin-4.0` one?

```
base_images:
  base:
    name: '4.0'
    namespace: ocp
    tag: base
```
Also note you can omit the `cluster` part.

Reply:
I followed this guide (https://docs.google.com/document/d/1SQ_qlkcplqhe8h6ONXdgBr7YUVbs4oRSj4ISl3gpLW4/edit#) and used some snippets from cluster-monitoring-operator and hive configuration as example.

Could you please point a newer/better example that I could check? In the meanwhile I will address your point below. Thanks

Reply:
That doc is fine, you probably just used some older versions of the configs to follow. Current version of e.g. the hive config looks up2date, you can follow it:

https://github.com/openshift/release/blob/master/ci-operator/config/openshift/hive/openshift-hive-master.yaml